CONTRIBUTING.md

```diff
@@ -329,6 +334,86 @@ If you already have the `refinebio-examples` docker image:
+- Checkout your newly created `publish` branch.
+- If you are certain that all changes in `staging` should be carried over to `master` (aka published to Github pages), then you can do a `git merge staging` into your new branch.
+- Now you can use the new branch to create the pull request to master as you would normally do.
+- Try to be as specific as possible about what PRs (and by relation, their commits) you are requesting to merge to `master`.
+
+This strategy allows you to resolve any merge conflicts in your new branch so we don't do a bad thing by committing conflict resolutions directly to `staging`.
+It also provides us with a "snapshot" if merges are continuing to happen to `staging` on other PRs -- this can make review hard if it keeps changing.
+
+**Scenario 2: Only some changes from staging should be published**  
+
+- Create a new branch from the most up-to-date `master` branch, call it `publish-mychanges` where `mychanges` is a short tag relevant to the changes.
+- Checkout your newly created `publish` branch.
+- For each commit that needs to be published, add it to your new branch by using `git cherry-pick <commit_id>`.
+Or in GitKraken, you can right click on the commit and choose `Cherry pick commit`.
+- Now you can use the new branch to create the pull request to `master` as you would normally do.
+- Try to be as specific as possible about what PRs (and by relation, their commits) you are requesting to merge to `master`.
+
+This strategy allows us to move forward changes from `staging` that are ready to be public facing even if other changes aren't ready (or if there isn't great timing for when the other changes will be ready).
+
+_Tips for getting commits ids_  
+GitKraken shows commits, but sometimes I find it hard to follow which commits belong to which branch.
+If you checkout `staging` and use `git log` you can see all the most recent commits for the `staging` branch.
+If you want to save it to a file for easy browsing and copy/pasting you can use this command `git log --pretty=format:'%h was %an, %ar, message: %s' > git.log`
+
+#### Publish a change, but quickly: direct merges to master, hotfixes
+
+In (hopefully rare) scenarios where something that has already been published is noted to be broken and should be addressed quickly, [hotfix branch](https://nvie.com/posts/a-successful-git-branching-model/#hotfix-branches) pull requests are allowed.
+These PRs should only be fairly small PRs and not anything that would require intense review.
+This more for situations where "this is broken and here's a fix".
+
+- Create a new branch from the most up-to-date `master` branch and call it `hotfix-mybug`, where where `mybug` is a short tag relevant to the changes.
+- Checkout your newly created `hotfix` branch.
+- Make the hotfix change.
+- Create the pull request to `master` as you would normally do.
```

Comment:
As one would normally create a pull request, the default would now be to merge to `staging` right?
I guess my question here is how do we switch between merging to `master` and merging to `staging` (mainly merging to `master` since `staging` is default as of now)?

Reply:
Good point. I think we can remove the "as you would normally do" here, as this is not "normal" for this repo, and you have to explicitly change the target, which is definitely not part of the typical process.

Reply:
I'll add some clarification here.